src/workers/VolumeLoadWorker.ts

```diff
@@ -77,6 +78,12 @@ const messageHandlers: { [T in WorkerMsgType]: MessageHandler<T> } = {
       }
     );
   },
+
+  [WorkerMsgType.SET_PREFETCH_PRIORITY_DIRECTIONS]: (directions) => {
+    // Silently does nothing if the loader isn't an `OMEZarrLoader`
+    (loader as OMEZarrLoader | undefined)?.setPrefetchPriorityDirections?.(directions);
```

Comment:
Would it be possible to have IVolumeLoader provide a default implementation that does nothing, and then have only OMEZarrLoader override the implementation so you don't have to do a type check/coercion here?